ui/pages/settings/security-tab/delete-metametrics-data-button/delete-metametrics-data-button.test.tsx

```diff
@@ -27,6 +28,9 @@ describe('DeleteMetaMetricsDataButton', () => {
   beforeEach(() => {
     useDispatchMock.mockReturnValue(mockDispatch);
     useSelectorMock.mockImplementation((selector) => {
+      if (selector === getParticipateInMetaMetrics) {
```

Comment:
Added due to https://github.com/MetaMask/metamask-extension/pull/29081/files#r1913396674